sdk/iOS/ZumoE2ETestApp/ZumoE2ETestApp/ZumoAppDelegate.m

```diff
@@ -12,6 +12,10 @@ @implementation ZumoAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     if([application respondsToSelector:@selector(registerForRemoteNotifications)]) {
+        
+        UIUserNotificationSettings *notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
+        [application registerUserNotificationSettings:notificationSettings];
+        
         [application registerForRemoteNotifications];
```

Comment:
shouldn't this now happen in the function triggered from registerUserNotificatinSettings being successful?